### DISCUSSION TITLE
`kitchen-sink` example on Vercel.

### DISCUSSION THREAD

**Discussion by jensmeindertsma** [vercel/turborepo#368] [Help] [answered]
I'm working to deploy the `kitchen-sink` example on Vercel ( using Yarn ).

I'm getting a ton of these errors:
```
> vercel-plugin-node.build: NowBuildError: error TS6053: File 'tsconfig/vite.json' not found.
```
```
> vercel-plugin-node.build: NowBuildError: error TS6053: File 'tsconfig/base.json' not found.
```

What should I do to resolve these errors? I believe this has do to with the Vercel builder resolving these files in `apps/admin/node_modules` after running `yarn` in `apps/admin`, while locally TypeScript resolves these files in `node_modules` after running `yarn` in the root. How do I make Vercel do the same as my local set up? It seems that I can't configure this properly as the Vercel builder is hardcoded to resolve the files in `apps/admin/node_modules`, while we need files from `apps/node_modules`?

**Comment by fivethreeo** [comment, marked as answer]
https://github.com/vercel/turborepo/issues/369
